monkey/monkey_island/cc/services/config.py

```diff
@@ -529,6 +532,15 @@ def _format_icmp_scan_from_flat_config(config: Dict):
 
         return formatted_icmp_scan_config
 
+    @staticmethod
+    def _format_fingerprinters_from_flat_config(config: Dict):
```

Comment:
At some point we should consider extracting these functions into separate "formatters" files.

Reply:
There's a separate task to make further changes to this code.